doc/asciidoc/linkprediction-adamic-adar.adoc

```diff
@@ -0,0 +1,105 @@
+[[algorithms-linkprediction-adamic-adar]]
+= The Adamic Adar algorithm
+
+[abstract]
+--
+This section describes the Adamic Adar Similarity algorithm in the Neo4j Graph Algorithms library.
+--
+
+// tag::introduction[]
+link:https://en.wikipedia.org/wiki/Adamic/Adar_index[Adamic Adar similarity] is a measure used to compute similarity between nodes based on their shared neighbors.
+// end::introduction[]
+
+
+[[algorithms-linkprediction-adamic-adar-context]]
+== History and explanation
+
+// tag::explanation[]
+
+Adamic Adar similarity was introduced in 2003 by Lada Adamic and Eytan Adar to https://www.semanticscholar.org/paper/Friends-and-neighbors-on-the-Web-Adamic-Adar/39348c10c90be968357e2a6b65d5e0e479307735[predict links in a social network^].
+It is computed using the following formula:
+
+image::adamic-adar-similarity.svg[role="middle"]
+
+where `N(u)` is the set of nodes adjacent to `u`.
+
+A value of 0 indicates that two nodes are dissimilar, while higher values indicate nodes are more similar.
+
+The library contains a function to calculate similarity between two nodes.
+
+// end::explanation[]
+
+[[algorithms-linkprediction-adamic-adar-usecase]]
+== Use-cases - when to use the Adamic Adar Similarity algorithm
+
+// tag::use-case[]
+We can use the Adamic Adar Similarity algorithm to work out the similarity between two nodes.
```

Comment:
We need to be careful with the consistency of naming this algorithm. It appears to be titled **The Adamic Adar algorithm**, but here it is titled **the Adamic Adar Similarity algorithm**. Above, it mentions _Adamic Adar similarity_, with a lower case "s".

I would recommend that we remove the "Similarity" from this example, and elsewhere in the page refer to only **The Adamic Adar algorithm** when we refer to "the algorithm".

Reply:
Have removed similarity for these as well